### PR TITLE
instr(metrics): Another tag for aggregation error

### PR DESCRIPTION
**Pull request by jjbayer**
Might help us understand https://github.com/getsentry/team-ingest/issues/227.

#skip-changelog

### CODE REVIEW

File: relay-metrics/src/statsd.rs [hunk 1]
```diff
@@ -184,3 +184,39 @@ impl GaugeMetric for MetricGauges {
...
+
+    if value.starts_with("s:spans/") {
+        return "s:spans/";
+    }
+    if value.starts_with("d:spans/") {
+        return "d:spans/";
+    }
+
+    "other"
+}
```
Comment (jjbayer) [inline relay-metrics/src/statsd.rs:222]: I brought this back temporarily. We can delete it again once we're done investigating.

File: relay-server/src/actors/processor.rs [hunk 2]
```diff
@@ -2604,17 +2609,9 @@ impl EnvelopeProcessorService {
...
-                        .aggregator_config_for(MetricNamespace::Transactions)
-                        .timestamp_range(),
-                ),
+                transaction_range: Some(agg.timestamp_range()),
                 max_name_and_unit_len: Some(
-                    self.inner
-                        .config
-                        .aggregator_config()
-                        .max_name_length
+                    agg.max_name_length
```
Comment (jjbayer) [inline relay-server/src/actors/processor.rs]: Drive-by fix: this was reading from the wrong aggregator config (without consequences because we rely on defaults anyway).

File: relay-server/src/actors/processor.rs [hunk 3]
```diff
@@ -2594,6 +2594,11 @@ impl EnvelopeProcessorService {
             .project_state
             .has_feature(Feature::SpanMetricsExtraction);
 
+        let agg = self
```
Comment (olksdr) [inline relay-server/src/actors/processor.rs]:
```suggestion
        let aggregator_config = self
```

I would really give it a full name

**Review by olksdr — APPROVED** [review]
